addons/mail/static/src/js/attachment_box.js

```diff
@@ -16,6 +17,8 @@ var AttachmentBox = Widget.extend({
         "click .o_attachment_delete_cross": "_onDeleteAttachment",
         "click .o_upload_attachments_button": "_onUploadAttachments",
         "change .o_chatter_attachment_form .o_form_binary_form": "_onAddAttachment",
+        'dragover .o_attachments_file_drop_zone': '_onFileDragover',
```

Comment:
I prefer naming DOM event handlers as `on<event type><name based on selector>()`.
In this case, this would be `_onDragoverFileDropZone()`